src/components/main/menu/NCSelector.jsx

```diff
@@ -0,0 +1,170 @@
+    const searchList = COUNCILS.filter((council) => searchFilter.test(council));
+    setFilteredCouncilList(searchList);
+    setSearchValue(e.target.value);
+  };
+
+  const handleSelectCouncil = (council) => {
+    const newSelectedCouncilList = { ...selectedCouncilList };
+
+    switch (council) {
+      case 'all': {
+        let value = true;
+
+        if (newSelectedCouncilList.all) {
+          newSelectedCouncilList.all = false;
+          value = false;
+        }
+
+        Object.keys(newSelectedCouncilList).forEach((c) => {
+          newSelectedCouncilList[c] = value;
+        });
+        break;
+      }
+      default:
+        newSelectedCouncilList.all = false;
+        newSelectedCouncilList[council] = !newSelectedCouncilList[council];
+        break;
+    }
+
+    const newNCList = Object.keys(newSelectedCouncilList).filter((c) => newSelectedCouncilList[c] && c !== 'all');
+
+    setSelectedCouncilList(newSelectedCouncilList);
+    updateNCList(newNCList);
+  };
+
+  return (
+    <div className="nc-selector" style={{ width: '349px' }}>
+      <div className="nc-title">
+        <p className="is-size-6" style={{ padding: '15px 0' }}>
+          <strong>
+            Neighborhood Council (NC) Selection
+          </strong>
+        </p>
+      </div>
+
+      <div className="nc-seach-list-wrapper">
+        <div className="nc-search">
+          <div className="field">
+            <div className="control has-icons-right">
+              <input
+                className="input"
+                type="text"
+                placeholder="Search"
+                style={{
+                  border: '1px solid #999999',
+                  borderRadius: '3px',
+                  boxSizing: 'border-box',
+
+                }}
+                value={searchValue}
+                onChange={handleSearch}
+              />
+              <span className="icon is-small is-right">
+                <i className="fas fa-search" />
+              </span>
+            </div>
+          </div>
+        </div>
+
+        <div
+          className="nc-list"
+          style={{
+            height: '231px',
+            overflowX: 'hidden',
+            msOverflowY: 'scroll',
+            padding: '10px 23px 10px 10px',
+            border: '1px solid #999999',
+            borderRadius: '3px',
+            boxSizing: 'border-box',
+          }}
+        >
+          <div className="level" style={selectRowStyle}>
+            <div className="level-left">
+              <div className="level-item">
+                <p className="is-size-7" style={selectRowTextStyle}>
+                  SELECT ALL
+                </p>
+              </div>
+            </div>
+            <div className="level-right">
+              <div className="level-item">
+                <Checkbox
+                  id="nc-select-all"
+                  handleClick={() => handleSelectCouncil('all')}
+                  checked={selectedCouncilList?.all ?? false}
```

Comment:
Just wanted to confirm - the new version of babel you installed transpiles this?

Reply:
Correct. You need to `npm i` before you can compile properly